src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
Your call, I didn't have time to get to it in the office today, and I won't have more time until 3 or 4 hours from now. If you want to I won't say no